packages/core/content-manager/admin/src/pages/EditView/components/FormInputs/Component/Input.tsx

```diff
@@ -20,7 +20,8 @@ interface ComponentInputProps
   extends Omit<Extract<EditFieldLayout, { type: 'component' }>, 'size' | 'hint'>,
     Pick<InputProps, 'hint'> {
   labelAction?: React.ReactNode;
-  renderInput?: (props: InputRendererProps) => React.ReactNode;
+  children: (name: InputRendererProps) => React.ReactNode;
+  layout: EditFieldLayout[][];
```

Comment:
Just because it's not clear from *only* looking at this file why we pass layout instead of finding it here, can we just write why we've done it? i.e. `to enable content history to do XYZ`